app/javascript/components/map-menu/components/sections/my-gfw/selectors.js

```diff
@@ -1,16 +1,40 @@
-import { createStructuredSelector } from 'reselect';
+import { createStructuredSelector, createSelector } from 'reselect';
 import isEmpty from 'lodash/isEmpty';
+import flatMap from 'lodash/flatMap';
+import uniq from 'lodash/uniq';
+import sortBy from 'lodash/sortBy';
 
 const selectLoading = state => state.mapMenu && state.mapMenu.loading;
 const selectLoggedIn = state => state.myGfw && !isEmpty(state.myGfw.data);
-const areas = state => state.areas && state.areas.data;
-const activeArea = state => state.areas && state.areas.activeArea;
-const location = state => state.location && state.location.payload;
+const selectAreas = state => state.areas && sortBy(state.areas.data, 'name');
+const selectLocation = state => state.location && state.location.payload;
+
+export const getActiveArea = createSelector(
+  [selectLocation, selectAreas],
+  (location, areas) => {
+    if (isEmpty(areas)) return null;
+
+    return areas.find(a => a.id === location.adm0);
+  }
+);
+
+export const getTags = createSelector([selectAreas], areas => {
```

Comment:
good idea moving these to a selectors file!

Reply:
:)